hawkins-worker.coffee

```diff
@@ -81,7 +81,7 @@ new Worker pushes, (push, processNext) ->
       github.repo(push.repository.full_name).status build.push.head_commit.id,
         state: status,
         target_url: "#{argv.base}/builds/#{buildKey}",
-        context: "Hawkins"
+        context: "ci"
```

Comment:
@rogercampos why is this a better context? The idea of those is to differentiate between quality services. `"Hawkins"` is a very specific one. The docs use `"continuous-integration/jenkins"` as an example, as well as `"ci"`.

Making it less specific could be useful in order to use different services for the same purpose, but this could mean we have 2 services stepping on each other. Other benefit could be to migrate easily to a different CI. Is this what you're after?